### DISCUSSION TITLE
Can't launch 1.17 Fabric Servers when Java 16 is installed

### DISCUSSION THREAD

**Discussion by Darkcraft951** [FabricMC/fabric-api#1694] [Player support]
So Fabric servers needs Java 16 to be opened, but when I have Java 16 installed on my PC, any modded(forge, fabric or anything else) servers in any version I try to open, it doesn't do anything. If anyone knows why i'd like to get some help

**Comment by warjort** [comment]
You aren't going to get much help with a question that is so vague and uninformative.
Explain what you have tried to do in detail and and show the results including error messages.

**Reply by Darkcraft951** [reply]
Well I don't have any error messages, nothing happens when I "open" the server when Java 16 is installed, but with Java 8 it just says I need Java 16 to be opened, and I tried uninstalling Java 16 and reinstalling it, that's all

**Comment by Technici4n** [comment]
When you run the server, try specifying the full path to your Java 16 install instead of just `java`. It's likely that just `java` points to an older install.
